Source/Experimental/FilterCallback/FilterParameter.h

```diff
@@ -258,6 +334,14 @@ class DataArrayPathParameter : public IFilterParameter
     DataArrayPathParameter(const std::string& humanLabel, const std::string& propertyName, const DataArrayPath &defaultValue, Category category, const SetterCallbackType& setterCallback, const GetterCallbackType& getterCallback, int groupIndex);
     
   private:
+    std::string m_HumanLabel = {};
+    std::string m_PropertyName = {};
+    Category m_Category = {};
+    int m_GroupIndex = {};
+    int m_DefaultValue = {};
+    double m_DefaultValue = {};
+    DataArrayPath m_DefaultValue = {};
```

Comment:
I know this is in experimental, but I don't think you can have multiple variables of the same name and different types.